2022/09.md

```diff
@@ -95,7 +95,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     |   | 2     | 60m     | [Array.fromAsync](https://github.com/tc39/proposal-array-from-async) for Stage 3 (slides TBD) | J. S. Choi |
     |   | 2     | 60m     | Set methods discussion ([slides](https://docs.google.com/presentation/d/1HCqPMsWiTtsn92gA3b1luVpnVHWVVR0iKaAE0marxkA)) | Kevin Gibbons |
     |   | 2     | 60m     | [Explicit Resource Management](https://github.com/tc39/proposal-explicit-resource-management) for Stage 3 ([slides](https://1drv.ms/p/s!AjgWTO11Fk-TkoEvgEld8y2bGCjTlw?e=ySkk5z), [spec](https://tc39.es/proposal-explicit-resource-management/)) | Ron Buckton |
-    |   | 1     | 15m     | [Iteration helpers as functions](https://github.com/rauschma/iterable) for stage 1 (slides TBD) | Axel Rauschmayer |
+    |   | –     | 15m     | Discussion: switching [iterator helpers](https://github.com/rauschma/iterable) to functions ([slides](https://speakerdeck.com/rauschma/iteration-helper-functions)) | Axel Rauschmayer |
```

Comment:
Stage should probably be either 0 (for a new proposal) or 2 (if you consider this to be a topic about iterator helpers, currently at stage 2). Also, it should then be ordered according to the proposal item ordering rules.

> Proposal-based agenda items should be sorted primarily by stage (descending), secondarily by timebox (ascending), and finally by insertion date.

Reply:
Thanks! Should be fixed now.